docs: mention how to use xdebug with drush 13+ (#6626)
ddev/ddev: docs/content/users/debugging-profiling/step-debugging.md

* You may need to open port 9003 in your firewall to allow Xdebug access. (See [Troubleshooting Xdebug](#troubleshooting-xdebug) below.)
* The IDE’s debug server port must be set to Xdebug’s default 9003, which is already the case in popular IDEs. If the unusual case that you have a port conflict, see [Using Xdebug on a Port Other than the Default 9003](#using-xdebug-on-a-port-other-than-the-default-9003) below.
* In the case of using running your IDE inside WSL2 (using WSLg) or with a proxy setup like JetBrains Gateway, you can set that with `ddev config global --xdebug-ide-location=wsl2`. If you're running your IDE with a proxy inside the web container, you can set that with `ddev config global --xdebug-ide-location=container`.

> * Drush 13+ disables Xdebug even when Xdebug is enabled on DDEV. As per [their docs](https://www.drush.org/13.x/commands/#xdebug) you can:

Reviewer: ```suggestion
* Drush 13+ disables Xdebug even when Xdebug is enabled on DDEV. As per [drush docs](https://www.drush.org/13.x/commands/#xdebug) you can:
```

    * Enable it for a single Drush command by running: `ddev drush --xdebug`

>     * Set `DRUSH_ALLOW_XDEBUG=1` [environment variable](../../extend/customization-extendibility.md#environment-variables-for-containers-and-services), allowing every drush call to be run with Xdebug if enabled. Not recommended as one will normally want to debug the site without that having a hit on drush commands.

Reviewer 2: ```suggestion
    * Set `DRUSH_ALLOW_XDEBUG=1` [environment variable](../../extend/customization-extendibility.md#environment-variables-for-containers-and-services), allowing every Drush call to be run with Xdebug when the PHP extension is enabled.
```

Reviewer 2: IMO we don't want to make a recommendation. Just let user do the easiest. They have been payingn the perf penalty for years and are fine.

Author: fine!
